ipa-core/src/protocol/ipa_prf/oprf_padding/mod.rs

```diff
@@ -130,6 +131,71 @@ pub trait Paddable {
         Self: Sized;
 }
 
+impl<BK, V> Paddable for IndistinguishableHybridReport<BK, V>
```

Comment:
This trait could be made better...

Reply:
can you take a look and see if this impl is cleaner?